packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -292,6 +383,8 @@ export async function generateBlogPosts(
     ignore: exclude,
   });
 
+  const authorList = await getAuthorList(contentPaths, options.authorListPath);
```

Comment:
It's not really a list right?
Not sure how to name it better.
`authors` can work but could also conflict with `metadatas.authors`
I'd use `authorsRegistry` or `authorsMap`, no strong opinion as long as it's not `authorsList`

Reply:
Yes, sure—initially I designed the API wrong (you can still see from my proposal), and when I really set my hands on it I realized it should be a map